### ISSUE TITLE
RFC: Delete Unrelated Icons

### ISSUE THREAD

**Issue by iamclaytonray** [iamclaytonray/devicon#4]
Currently, there are a few icons that seem like they'd be better coming from another icon library. For example, Twitter. We should keep this library as minimal, simple, and tightly coupled to developer/design/language icons as much as possible. What are your thoughts?

**Comment by MattIPv4** [comment]
If this is to be done, each icon to be removed should have its own issue with the proposal to remove and a discussion about whether it should be removed before it is.

**Comment by iamclaytonray** [comment]
I agree. This is more of an RFC to see if we should do that or just leave them be
